getting_started/toy_shop/packages.yml

```diff
@@ -2,6 +2,6 @@ packages:
   - package: dbt-labs/dbt_utils
     version: [">=0.7.0", "<0.9.0"]
 
-  - package: re-data/re_data
-    version: [">=0.7.0", "<0.8.0"]
+  - git: https://github.com/re-data/dbt-re-data.git
```

Comment:
@dejii I assume before merging you are changing that? Btw I pushed to dbt hub / so it should show up in 30 minutes there and be available to use

Reply:
Okay, I would be on the look. @mateuszklimek